src/resource.c

```diff
@@ -67,6 +69,7 @@ typedef struct {
     RESOURCE_CONFIG_COMMON;
     char                    *group;      /* audio group */
     pid_t                    pid;        /* PID of the streaming component */
+    char                    *app_id;     /* application id of the streaming component */
     char                   *stream;     /* pulseaudio stream name */
```

Comment:
Nitpick, maybe indentation of the second column of this line could be fixed to make it consistent.